### PR TITLE
Vars and SSM Params for EB Proactive Notifications

### PR DESCRIPTION
**Pull request by david-dreyer**

### CODE REVIEW

File: constant_vars.tf [hunk 1]
```diff
@@ -216,4 +216,15 @@ variable "buckets_to_protect" {
   description = "Enter any pre-existing buckets that you would like to automatically enable event-based protection on. Bucket names must be separated by commas (e.g. bucket1,bucket2,bucket3). Protected buckets can be managed after deployment in the CSS Console."
   type        = string
   default     = ""
-}
\ No newline at end of file
+}
+
+variable "eventbridge_notifications_enabled" {
+  type        = bool
+  description = false
```
Comment (sh-dan) [inline constant_vars.tf:223]: should be default

File: ssm_main.tf [hunk 2]
```diff
@@ -284,3 +284,21 @@ resource "aws_ssm_parameter" "CloudTrailLakeChannelArnParameter" {
...
+  }
+}
+
+resource "aws_ssm_parameter" "EventBridgeNotificationsBusNameParameter" {
+  name  = "/${var.parameter_prefix}-${aws_appconfig_application.AppConfigAgentApplication.id}/Config/EventBridgeNotificationsBusName"
+  type  = "String"
+  value = var.eventbridge_notifications_bus_name
+  lifecycle {
+    ignore_changes = [value]
+  }
```
Comment (sh-dan) [inline ssm_main.tf:303]: If these two can't be modified by the app, then "source of truth" on what it should be is TF. 
In this case we can remove the lifecycle / ignore block so any TF changes with change the param.

File: constant_vars.tf (hunk 1 above)
Comment (david-dreyer) [reply constant_vars.tf:223]: Haha obviously

File: ssm_main.tf (hunk 2 above)
Comment (david-dreyer) [reply ssm_main.tf:303]: Yup, they can be changed by the App

File: constant_vars.tf (hunk 1 above)
Comment (david-dreyer) [inline constant_vars.tf:223]:
```suggestion
  default = false
```

**Review by sh-dan — APPROVED** [review]

**Pull request merged**